src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_NUMBER = 20;
+    public static final int FIGUR_NUMBER = 5;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        // Square getRandomFigure
+        String colorSquare = new ColorSupplier().getRandomColor();
```

Comment:
ColorSupplier should not be local
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/4d1ebc70-c724-4109-aac0-169bea07dd97)